src/master-playlist-controller.js

```diff
@@ -472,6 +472,14 @@ export class MasterPlaylistController extends videojs.EventTarget {
 
     this.masterPlaylistLoader_.on('playlistunchanged', () => {
       const updatedPlaylist = this.masterPlaylistLoader_.media();
+
+      // ignore unchanged playlists that have already been
+      // excluded for not-changing. We likely just have a really slowly updating
+      // playlist.
+      if (updatedPlaylist.lastExcludeReason_ === 'playlist-unchanged') {
```

Comment:
Is it also worth checking the last `excludeUntil` date to see if we're passed the point where the exclusion should be considered?

Reply:
🤷🏻 It might be but I think that `playlist-unchanged` excludes are not likely to stop happening no matter how much time has passed.